app/models/user.rb

```diff
@@ -26,15 +26,15 @@ def gravity_user
   end
 
   def name
-    self[:name] || gravity_user.try(:name)
+    gravity_user.try(:name)
   end
```

Comment:
do we want to get user name from gravity, even if user provides a (potentially different) name in contact information?

Reply:
https://github.com/artsy/convection/blob/dd1a2571530faaca4a6d0ed50cb9bb52bcfd7b0a/app/models/submission.rb#L192-L194

@AlicanAkyuz, no we do not want to get user details from Gravity if a user provided contact information in the 3rd step of the new submission flow. The change above is what will make that happen for submissions.

Reply:
I'm trying to get info from the [submission](https://github.com/artsy/convection/blob/dd1a2571530faaca4a6d0ed50cb9bb52bcfd7b0a/app/models/submission.rb#L193), but if it's not there, then I call the name method from the user, which takes info from gravity

Reply:
understood, thank you.